src/SWP/Bundle/TemplateEngineBundle/Command/ThemeGenerateCommand.php

```diff
@@ -0,0 +1,265 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Fixtures Bundle.
+ *
+ * Copyright 2016 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+namespace SWP\Bundle\TemplateEngineBundle\Command;
+
+use SWP\Bundle\CoreBundle\Document\Tenant;
+use SWP\Component\MultiTenancy\Exception\OrganizationNotFoundException;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Filesystem\Filesystem;
+
+class ThemeGenerateCommand extends ContainerAwareCommand
+{
+    const THEMES_DIR = 'themes';
```

Comment:
yeap, provided as a default value in `TenantableConfigurationSourceFactory` class so themes dir is configurable via theme bundle config. The same in case `theme.json`.